tools/crm_mon.c

```diff
@@ -1085,7 +1079,7 @@ main(int argc, char **argv)
 #endif
             } else {
                 if (output_format == mon_output_html) {
-                    print_as(output_format, "Writing html to %s ...\n", output_filename);
+                    printf("Writing html to %s ...\n", args->output_dest);
```

Comment:
Hm, yes probably.  I don't think output_dest can ever be stdout for HTML at the moment but it's probably best to be ready.

Reply:
Wouldn't --output-as=html --output-to=- do it?

Reply:
It should, but it currently does not:

```
clumens@localhost:~/src/pacemaker$ CIB_file=cts/scheduler/anon-instance-pending.xml tools/crm_mon --output-as html --output-to -
lt-crm_mon: Missing argument for --output-to
Usage:
  lt-crm_mon [OPTION]
```

So I guess I need to figure that out.  It just seemed a little out of the way at the moment, since people aren't used to these options yet.  I've fixed it anyway.

Reply:
#1889 should fix that up.